decision_making.md

```diff
@@ -0,0 +1,45 @@
+# Decision Making Guide
+
+This document describes how Jupyter’s governing bodies (Board of Directors, Software Steering Council, Software Projects, Working Groups) make decisions. Because individuals often work across multiple decision making groups, all Jupyter governing bodies are expected to follow this approach.
+
+We seek to honor the principles of collaboration, inclusive participation, and responsive decision making. Some aspects of this decision making guide are required of all teams. Others are provided as recommendations and are optional. We have clearly noted the optional aspects of decision making below.
+
+Finally, the Jupyter Board of Directors may either intervene, or be called upon by members of a team, in the event that issues arise in the decision making process. This includes, but is not limited to: process ambiguity, violations of the decision making process, mitigating circumstances that require process exceptions, etc.
```

Comment:
Today in the governance working group call, we discussed this suggestion and arrived at:

- We should keep the language as is because this document is describing the new governance model.
- This suggestion is to mitigate ambiguity in the transition period, which we should try to keep as brief as possible, but mixing the current SC with the nascent BoD in this document might create confusion.
- We are adding a transition section to the end of this document to help move the process along. That section is designed to be removed once the new governance model is approved.